src/SimpleGraphs/Generators/staticgraphs.jl

```diff
@@ -270,6 +270,27 @@ function _binary_tree(k::T) where {T<:Integer}
     return SimpleGraph{T}(ne, fadjlist)
 end
 
+function _binary_tree_digraph(k::T) where {T<:Integer}
+    k <= 0 && return SimpleDiGraph{T}(0)
```

Comment:
I just suggest this because this is Seth and I know optimization tricks make you happy:
```julia
k <= 1 && return SimpleDiGraph{T}(k == 1)
```
The `{T}` will upcast the boolean in appropriate integer. Feel free to ignore this :D

Reply:
It's a little TOO magic :)